exercises/practice/linked-list/linked_list.go

```diff
@@ -15,19 +15,19 @@ func (n *Node) Prev() *Node {
 	panic("Please implement the Prev function")
 }
 
-func (l *List) PushFront(v interface{}) {
+func (l *List) Unshift(v interface{}) {
 	panic("Please implement the PushFront function")
 }
 
-func (l *List) PushBack(v interface{}) {
+func (l *List) Push(v interface{}) {
 	panic("Please implement the PushBack function")
 }
 
-func (l *List) PopFront() (interface{}, error) {
+func (l *List) Unshift() (interface{}, error) {
 	panic("Please implement the PopFront function")
 }
```

Comment:
```suggestion
func (l *List) Shift() (interface{}, error) {
	panic("Please implement the Shift function")
}
```

The Unshift function is repeated here. I think you meant Shift.